spec/models/datatables/teams_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
Nitpicking, but... rename this file to `spec/models/datatables/datatables_team_spec.rb`.